pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -124,11 +124,17 @@ func (b *HybridBotanist) DeployKubeAPIServer() error {
 		"advertiseAddress":      loadBalancerIP,
 		"cloudProvider":         b.ShootCloudBotanist.GetCloudProviderName(),
 		"kubernetesVersion":     b.Shoot.Info.Spec.Kubernetes.Version,
-		"podNetwork":            b.Shoot.GetPodNetwork(),
-		"serviceNetwork":        b.Shoot.GetServiceNetwork(),
-		"nodeNetwork":           b.Shoot.GetNodeNetwork(),
-		"securePort":            443,
-		"probeCredentials":      utils.EncodeBase64([]byte(fmt.Sprintf("%s:%s", b.Secrets["kubecfg"].Data["username"], b.Secrets["kubecfg"].Data["password"]))),
+		"shootNetworks": map[string]interface{}{
+			"service": b.Shoot.GetServiceNetwork(),
+		},
+		"seedNetworks": map[string]interface{}{
+			"service": b.Seed.Info.Spec.Networks.Services,
+			"pod":     b.Seed.Info.Spec.Networks.Pods,
+			"node":    b.Seed.Info.Spec.Networks.Nodes,
+		},
+		"cloudProviderMetadataServiceIP": b.SeedCloudBotanist.GetCloudProviderMetadataServiceIP(),
```

Comment:
Could we maintain the metadata service ips inside Chart and depending on the value of ``cloudProvider`` key in the Charts ``values.yaml``, insert the proper the ip? Then there were no need to maintain it in the botanists code.